Resources/views/column/column.html.twig

```diff
@@ -23,11 +23,6 @@
         {% if column.width is not same as(null) %}
             "width": "{{ column.width }}",
         {% endif %}
-        {% block title %}
```

Comment:
Why removing this block?

Reply:
Hmm... It seems it's a mistake... A try I forgot to restore...

Sorry, my next commit will fix it.